service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java

```diff
@@ -67,23 +99,207 @@
     @MockBean
     private HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration;
 
+    @MockBean
+    private RepresentativesInbox representativesInbox;
```

Comment:
Could you update this test to not be reliant on spring,
the only mocks you should need are these which result in the following field declaration and class annotation
```java
@ExtendWith(MockitoExtension.class)
class AdditionalApplicationsUploadedEventHandlerTest {

    @Mock
    private RequestData requestData;
    @Mock
    private NotificationService notificationService;
    @Mock
    private HmctsAdminNotificationHandler adminNotificationHandler;
    @Mock
    private AdditionalApplicationsUploadedEmailContentProvider additionalApplicationsUploadedEmailContentProvider;
    @Mock
    private InboxLookupService inboxLookupService;
    @Mock
    private RepresentativesInbox representativesInbox;
    @Mock
    private OtherRecipientsInbox otherRecipientsInbox;
    @Mock
    private RepresentativeNotificationService representativeNotificationService;
    @Mock
    private SendDocumentService sendDocumentService;
    @Mock
    private FeatureToggleService featureToggleService;

    @InjectMocks
    private AdditionalApplicationsUploadedEventHandler additionalApplicationsUploadedEventHandler;
```

Also `additionalApplicationsUploadedEventHandler` could be renamed to `underTest` to make lines a bit shorter :)

This will also reveal that not everything that is present in the tests is correct (along the same lines as my comment in the handler int test)